client/src/tutorials/FaucetSeahorse/pages/2.md

```diff
@@ -0,0 +1,43 @@
+# Initialize our faucet
+
+Our program assumes the Bitcorn token we are distributing already exists. This means when we come to testing the program we will have to create the token ourselves which we'll do from the command line. To initialize our faucet we provide four arguments -- a `Signer`, a `TokenMint` account, an empty instance of the `BitcornFaucet` account and an empty instance of a `TokenAccount`.
+
+A token mint account is a place that stores the essential information related to a token such as the total supply and the creator of the token. Each token has a single mint account. For a user to hold a token, they must first create a token account for that token. Each user can only have one token account per token and that account stores the data for how many tokens the user holds.
+
+Our faucet is initialized with the seeds `['mint', mint]`, a string and the address of the token mint account. We also store the bump used to generate the PDA in a variable.
+
+```py
+@instruction
+def init_faucet(
+    signer: Signer,
+    mint: TokenMint,
+    faucet: Empty[BitcornFaucet],
+    faucet_account: Empty[TokenAccount]):
+
+  bump = faucet.bump()
+
+  faucet = faucet.init(
+    payer = signer,
+    seeds = ['mint', mint]
+  )
+```
+
+For the most part Seahorse abstracts away our need to deal with 'bumps' or 'bump seeds'. A bump is a `u8` unsigned integer (0-255) that is used as part of the process to create PDAs. Seahorse deals with this process so you don't have to. If you want to know more about bumps, you can read more [here](https://solanacookbook.com/core-concepts/pdas.html#facts). It's a concept you will need to understand at some point in your Solana developer journey. When building with PDAs it is common to store the bump seed in the account data itself as we do with our faucet.
+
+Finally we also store the mint and owner addresses in the relevant fields of our faucet's account.
+
+``py
+faucet_account.init(
+payer = signer,
+seeds = ["token-seed", mint],
+mint = mint,
+authority = faucet,
```

Comment:
I think these lines need indentation

Reply:
Is this resolved?

Reply:
Running 'yarn format' changes it back everytime